<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chatbot</class>
 <widget class="QDialog" name="chatbot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>684</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);
border:4px solid rgb(178,225,155);</string>
  </property>
  <widget class="QTextEdit" name="chatDisplayTextEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>621</width>
     <height>271</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border:4px solid rgb(178,225,155);</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="SendButton">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>390</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">ont:10pt&quot;Centaur&quot;;
color:rgb(0,170,0);
border-radius:10px;
border:4px solid rgb(178,225,155);</string>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="messageTextEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>390</y>
     <width>501</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit{
	font: 9pt &quot;Terminal&quot;;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>621</width>
     <height>31</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
    
	
	font: 700 16pt &quot;Tahoma&quot;;
   background-color: rgb(178,225,155);
}
</string>
   </property>
   <property name="text">
    <string>SMART FARM CHATBOT</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
